app/lib/ui/screens/chat/chat_view.dart

```diff
@@ -85,22 +83,43 @@ class ChatView extends StatelessWidget {
                     child: ListView.separated(
                       padding: _chatListPadding,
                       separatorBuilder: (_, __) => _sizedBox,
+                      reverse: true,
                       physics: const NeverScrollableScrollPhysics(),
                       shrinkWrap: true,
                       itemCount: message.value.length,
-                      itemBuilder: (BuildContext context, int index) =>
-                          Container(
-                        padding: _messageContentPadding,
-                        decoration: BoxDecoration(
-                          color: isCurrentUser
-                              ? Theme.of(context).colorScheme.primaryContainer
-                              : Theme.of(context).colorScheme.outlineVariant,
-                          borderRadius: _messageBorderRadius,
-                        ),
-                        child: Text(
-                          message.value[index].content,
-                        ),
-                      ),
+                      itemBuilder: (BuildContext context, int index) {
+                        final ChatModelMessage msg = message.value[index];
+                        return GestureDetector(
+                          onLongPress: () => _showDeleteDialog(context, msg),
```

Comment:
**suggestion:** Check author id. currently we can delete any messages, even from our chat partner :D